.github/workflows/cross-chain-arbitrum.yml

```diff
@@ -0,0 +1,187 @@
+name: Cross-chain Arbitrum module
+
+# There are a couple of places where we hardcode the environment name to
+# "arbitrumGoerli". As of adding this config file, there is no easy way of passing
+# an additional parameter to this workflow like ${{ github.event.inputs.environment-l2 }}
+# and would require a rebuild of the custom GH actions used to handle the
+# triggering of workflows in the CI flow.
```

Comment:
Ah, it will be added in the follow up PR where we re-add the missing publishing and deployment jobs. Removed here [8c58d02](https://github.com/keep-network/tbtc-v2/pull/554/commits/8c58d021af1a31336af80fba6b4bddc0fb113de5)